browser/Makefile.am

```diff
@@ -1111,7 +1110,7 @@ pot:
 		src/app/DocEvents.ts \
 		src/app/GraphicSelectionMiddleware.ts \
 		src/errormessages.js \
-		src/layer/tile/CanvasTileWorker.js \
+		src/layer/tile/CanvasTileWorker.js.m4 \
```

Comment:
We do not need neither src/layer/tile/CanvasTileWorker.js.m4 nor src/layer/tile/CanvasTileWorker.js in the pot section, because they do not contain localizable strings.